editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4333,8 +4343,37 @@ void CanvasItemEditor::_set_anchors_preset(Control::LayoutPreset p_preset) {
 	undo_redo->commit_action();
 }
 
+float CanvasItemEditor::_get_next_zoom_value(int increment_count) const {
```

Comment:
Last nitpick and it's good to merge: we prefix function parameters with `p_`, so this should be `p_increment_count` (same in the header).

Reply:
Should be fixed now :)